File: Problem_Solving_Algorithms/compare_the_triplets.js
function compareTriplets(alice, bob) {
    let aliceScore = 0;
    let bobScore = 0
    for (let i = 0; i < alice.length; i++) {
        for (let j = 0; j < bob.length; j++) {
            if (alice[i] > bob[i]) {
                aliceScore++;
            } else if (alice[i] < bob[j]) {
                bobScore++;
            }
            i++;
        }
    }
    return [aliceScore, bobScore];
}

File: Problem_Solving_Algorithms/compare_the_triplets.rb
def compareTriplets(alice, bob)
    alice_score = 0
    bob_score = 0

    alice.zip(bob).each do |alice_val, bob_val|
        if alice_val > bob_val
          alice_score += 1
        elsif alice_val < bob_val
          bob_score += 1 
        end
    end

    [alice_score, bob_score]

end